bindings/wasm/test/examples/gyroid-module.mjs

```diff
@@ -0,0 +1,65 @@
+// Recreates Modular Gyroid Puzzle by Emmett Lalish:
+// https://www.thingiverse.com/thing:25477. This sample demonstrates the
+// use of a Signed Distance Function (SDF) to create smooth, complex
+// manifolds.
+import {vec3} from 'gl-matrix';
```

Comment:
Now that you have imports working, can we make gl-matrix just an import, so manifold-cad doesn't need to know about it anymore? Would make a good demonstration of the work you've done.